source/docs/03-metadata.xml

```diff
@@ -124,6 +125,18 @@
                   <item>This concept is covered in section <ptr target="#headerusecases"/>.</item>
                   <item>A <hi rend="italic">revision history</hi>, tagged <gi scheme="MEI">revisionDesc</gi>, which allows the encoder to provide a history of changes made during the development of the electronic text. The revision history is important for version control and for resolving questions about the history of a file. The MEI elements used to encode the revision description are described in section <ptr target="#headerRevisionDescription"/>.</item>
                </list>
+               <div xml:id="nonfrbrsourcedescription" type="div3">
+                  <head>Non-FRBR Source Description</head>
+                  <p>MEI allows two methodologies to be used when doing metadata capture and source description. The <abbr>FRBR</abbr> methodology (<ptr target="#FRBR"/>) gives encoders a mechanism for building detailed descriptions of bibliographic entities. Most bibliographic-based projects should favor the FRBR method.</p>
+                  <p>However, there are cases where detailed bibliographic analysis at the level required by the FRBR module is not possible or desirable. These may include situations such as retro-conversion of non-FRBR catalogues, or capturing the output of notation encoding software where only minimal source description capabilities are available. In these cases, encoders may capture metadata in the MEI header without using FRBR encoding entities and instead choose a simplified methodology for source description.</p>
+                  <p>The <gi scheme="MEI">sourceDesc</gi> element as a child of the <gi scheme="MEI">fileDesc</gi> element may be used to capture simple bibliographic descriptions. An example is given below.</p>
```

Comment:
@pe-ro @riedde Is the idea to always use the new `biblFull` element for this purpose. Maybe `biblFull` should be mentioned here, so this would fix #1619 as well?

Reply:
`<source>` may use `<bibl>, `<biblStruct>`, or `<biblFull>`.  The first two permit bibliographic *citation* for the souce, while `<biblFull>` allows bibliographic *description*.  In other words, `<bibl>` and `<biblStruct>` are not as fully-featured as `<biblFull>`.

I think you're right that it would be advantageous to modify the text here.